spill/snake.ino

```diff
@@ -9,10 +9,14 @@ const int YPin = 34;
 
 const int w = tft.width(); // 320
 const int h = tft.height(); // 480
-char currentDirection;
+char currentDirection = 'R';
 int xPos = 0;
 int yPos = 0;
 
+int positions[1][2] = [[0,0]]; // a list with all cordinates.
```

Comment:
positions is a one element list containing a two element list containing integers. Initialized to what?

Why is it only one big? What was the point of this?